### PR TITLE
Add show-version-diff utility

### PR DESCRIPTION
**Pull request by mheap**
### Summary
Show the changes between all versions of a single file using `diff`.

The following command shows all changes to `deck.md`, ignoring the `pre-1.7` folder.

```bash
node run.js --source ../app/deck/1.11.x/reference/deck.md --ignore "pre-1.7"
```

https://user-images.githubusercontent.com/59130/159183785-6262dad2-3470-4c98-9267-5bf65b20fb70.mp4

### Reason
To make it easier to see differences between versions whilst combining files. Useful once #3765 is merged

### Testing
Clone the branch, run the command above and take a look at the output

### CODE REVIEW

File: show-version-diff/run.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    ]);
+    console.log(
+      `No differences between ${currentPath} and ${nextPath}\n-----------------\n`
+    );
+  } catch (e) {
+    // Diff always returns exit code 1, so we have to catch
+    console.log(
+      `##########################################\n${currentPath}\n${nextPath}\n##########################################`
+    );
+    if (e.stdout.length > 0) {
```
Comment (lena-larionova) [inline show-version-diff/run.js:127]: I keep getting the following error when I try to run this:

Screen Shot 2022-03-21 at 10 53 30 AM

I tried this with a bunch of different files, including the decK one that your provided in the example. Not sure what the difference is here...

File: show-version-diff/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Show Version Diff
+
+Show the changes between all versions of a single file using `diff`.
+
+The following command shows all changes to `deck.md`, ignoring the `pre-1.7` folder.
```
Comment (lena-larionova) [inline show-version-diff/README.md:5]:
```suggestion
Run the diff script from the `show-version-diff` folder. The following 
command shows all changes to `deck.md`, ignoring the `pre-1.7` folder:
```
Useful to mention that you have to be in this folder to run it. I ran into that issue on my first attempt to test. Seems like common sense in hindsight, but it wasn't my first instinct.

**Review by lena-larionova — COMMENTED** [review]
Diffs between files that have no changes were successful:

Screen Shot 2022-03-21 at 11 02 05 AM

Diffs between changed files, however, keep failing for me. Am I missing some package that I need to have installed locally to run this?

File: show-version-diff/run.js (hunk 1 above)
Comment (mheap) [reply show-version-diff/run.js:127]: It uses `git diff` under the hood, so it should be portable. Might be one to hop on a call and poke around to see what's going on

Comment (lena-larionova) [reply show-version-diff/run.js:127]: Messed around with this a bunch, still no luck (though a regular `git diff` works fine). I think line 127 isn't reading from stdout for me, for whatever reason. We can do a short call tomorrow/later this week to figure this one out.

**Review by lena-larionova — APPROVED** [review]
Works locally, LGTM

**Pull request merged**